src/server/graphql/queries/organization.js

```diff
@@ -20,7 +20,8 @@ export default {
     const {orgUsers} = org;
     const myOrgUser = orgUsers.find((user) => user.id === viewerId);
     if (!myOrgUser || myOrgUser.role !== BILLING_LEADER) {
-      throw new Error('Must be a billing leader');
+      // silently fail
```

Comment:
Why silently fail?

Reply:
the alternative is to wrap the organization in a payload:
```
orgPayload {
  error {
    message
  }
  organization
}
```
that's not great, especially because the only reason it would fail is because someone was doing something nefarious. i don't think those folks deserve an error message.